assembl/static2/js/app/utils/translations.js

```diff
@@ -450,6 +450,12 @@ const Translations = {
           editTooltip: "Modifier",
           deleteTooltip: "Supprimer"
         },
+        sentiment: {
+          like: "Ça m'inspire !",
+          dislike: "Ça me refroidit !",
```

Comment:
Is this the translation we chose 😂 ?

Reply:
@MilanBarande It's the text chosen by the top :)